huxley/core/models.py

```diff
@@ -123,6 +124,8 @@ class School(models.Model):
     fees_owed = models.DecimalField(max_digits=6, decimal_places=2, default=Decimal('0.00'))
     fees_paid = models.DecimalField(max_digits=6, decimal_places=2, default=Decimal('0.00'))
 
+    modified_at = models.DateTimeField(default=timezone.now)
+
```

Comment:
auto_now doesn't play nicely here because it needs to adjust when update() is called on schools, and the default update() method in django does not call save(), which is the trigger for auto_now